doc/classes/Quaternion.xml

```diff
@@ -4,19 +4,24 @@
 		A unit quaternion used for representing 3D rotations.
 	</brief_description>
 	<description>
-		Quaternions are similar to [Basis], which implements the matrix representation of rotations. Unlike [Basis], which stores rotation, scale, and shearing, quaternions only store rotation.
-		Quaternions can be parametrized using both an axis-angle pair or Euler angles. Due to their compactness and the way they are stored in memory, certain operations (obtaining axis-angle and performing SLERP, in particular) are more efficient and robust against floating-point errors.
-		[b]Note:[/b] Quaternions need to be normalized before being used for rotation.
+		The [Quaternion] built-in [Variant] type represents rotation in the form of a [url=https://en.wikipedia.org/wiki/Quaternions_and_spatial_rotation]JPL convention quaternion[/url]. While they are very often equivalent to [Basis], quaternions can [i]only[/i] store rotation.
```

Comment:
I see that you and @fire have mentioned over the past year that Godot uses JPL, but the Wikipedia page defines JPL as `i * j = -k`, but this is not the behavior of Godot:

```
	var a := Quaternion(1, 0, 0, 0)
	var b := Quaternion(0, 1, 0, 0)
	print(a * b) # (0, 0, 1, 0), so i * j = k
```

Where does this idea that Godot uses JPL come from? As far as I can tell, it does not.

Reply:
Does it mean that the very original issue, https://github.com/godotengine/godot-docs/issues/6271, is completely incorrect?

Reply:
Yes. @fire said JPL in that issue but I objected to it at the time because as far as I could tell that was incorrect and I asked for clarification and I do not recall receiving any. I believe that issue is just completely incorrect indeed.